{"total_count": 1, "incomplete_results": false, "items": [{"url": "https://api.github.com/repos/demurgos/py-pathmatch/issues/2", "repository_url": "https://api.github.com/repos/demurgos/py-pathmatch", "labels_url": "https://api.github.com/repos/demurgos/py-pathmatch/issues/2/labels{/name}", "comments_url": "https://api.github.com/repos/demurgos/py-pathmatch/issues/2/comments", "events_url": "https://api.github.com/repos/demurgos/py-pathmatch/issues/2/events", "html_url": "https://github.com/demurgos/py-pathmatch/issues/2", "id": 611327932, "node_id": "MDU6SXNzdWU2MTEzMjc5MzI=", "number": 2, "title": "Deprecation warning due to invalid escape sequences.", "user": {"login": "tirkarthi", "id": 3972343, "node_id": "MDQ6VXNlcjM5NzIzNDM=", "avatar_url": "https://avatars3.githubusercontent.com/u/3972343?v=4", "gravatar_id": "", "url": "https://api.github.com/users/tirkarthi", "html_url": "https://github.com/tirkarthi", "followers_url": "https://api.github.com/users/tirkarthi/followers", "following_url": "https://api.github.com/users/tirkarthi/following{/other_user}", "gists_url": "https://api.github.com/users/tirkarthi/gists{/gist_id}", "starred_url": "https://api.github.com/users/tirkarthi/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/tirkarthi/subscriptions", "organizations_url": "https://api.github.com/users/tirkarthi/orgs", "repos_url": "https://api.github.com/users/tirkarthi/repos", "events_url": "https://api.github.com/users/tirkarthi/events{/privacy}", "received_events_url": "https://api.github.com/users/tirkarthi/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 2, "created_at": "2020-05-03T04:10:42Z", "updated_at": "2020-05-03T23:30:32Z", "closed_at": "2020-05-03T23:30:32Z", "author_association": "NONE", "active_lock_reason": null, "body": "Deprecation warning due to invalid escape sequences. Using raw strings or escaping them again helps in resolving this. Check https://github.com/asottile/pyupgrade/ for automatic fix of this.\r\n\r\n```\r\nfind . -iname '*.py' | grep -Ev 'rdf4|doc|benchmark|tool' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}\r\n./pathmatch/wildmatch.py:52: DeprecationWarning: invalid escape sequence \\A\r\n  _PY_IMPOSSIBLE_MATCH = u'(?:a\\A)'\r\n```", "performed_via_github_app": null, "score": 1.0}]}